Android/app/src/main/java/io/github/project_travel_mate/HomeFragment.java

```diff
@@ -53,22 +53,30 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         mFriendsView.setOnClickListener(v1 -> {
             Fragment friendsFragment = new MyFriendsFragment();
             fragmentManager.beginTransaction().replace(R.id.parent_home,
-                    friendsFragment).commit();
+                    friendsFragment)
```

Comment:
there is being too much code repitition. Move 
```
fragmentManager.beginTransaction().replace(R.id.parent_home,
                    tripsFragment).commit();	                    tripsFragment)
                    .addToBackStack(null)
                    .commit();
```

to a new function. You can add fragment parameter to that function

Reply:
Okay sure @Swati4star . But please take a look here https://github.com/project-travel-mate/Travel-Mate/issues/614#issuecomment-484038568